<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IcePlayer</class>
 <widget class="QWidget" name="IcePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IcePlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(217, 227, 236)</string>
  </property>
  <widget class="QLabel" name="picLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>45</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Resources/default_album.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="iceLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>55</y>
     <width>220</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="musicianLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>85</y>
     <width>220</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>15</height>
    </size>
   </property>
   <property name="text">
    <string>Musician</string>
   </property>
  </widget>
  <widget class="QLabel" name="albumLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>115</y>
     <width>220</width>
     <height>15</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>15</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>220</width>
     <height>15</height>
    </size>
   </property>
   <property name="text">
    <string>Album</string>
   </property>
  </widget>
  <widget class="QSlider" name="playSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal
{
border:0px;
height:4px;
}

QSlider::sub-page:horizontal
{
background:#0096ff;
}

QSlider::add-page:horizontal
{
background:lightgray;
}

QSlider::handle:horizontal
{
background:white;
width:10px;
border:#51b5fb 10px;
border-radius:5px;
margin:-3px 0px -3px 0px;
}</string>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="playlistTable">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>240</y>
     <width>350</width>
     <height>320</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QScrollBar
{
background:transparent;
 width: 5px;
}

QScrollBar::handle
{
background:lightgray;
 border:2px solid transparent; 
border-radius:5px;
}
		
QScrollBar::handle:hover
{
background:gray;
}

QScrollBar::sub-line
{
background:transparent;
}

QScrollBar::add-line
{
background:transparent;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-style:solid;
border-width:0px;
border-color:rgb(255,255,255);
background:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resources/add.png</normaloff>:/Resources/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="modeButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-style:solid;
border-width:0px;
border-color:rgb(255,255,255);
background:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resources/mode.png</normaloff>:/Resources/mode.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lastButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>185</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-style:solid;
border-width:0px;
background:transparent;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resources/last.png</normaloff>:/Resources/last.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>185</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255,255,255,0);
border-style:solid;
border-width:0px;
border-color:rgba(255,255,255,0);
background:transparent;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resources/next.png</normaloff>:/Resources/next.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lyricButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-style:solid;
border-width:0px;
border-color:rgb(255,255,255);
background:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resources/lyr.png</normaloff>:/Resources/lyr.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>175</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-style:solid;
border-width:0px;
border-color:rgb(255,255,255);
background:transparent;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resources/play.png</normaloff>
     <normalon>:/Resources/pause.png</normalon>
     <disabledoff>:/Resources/play.png</disabledoff>
     <disabledon>:/Resources/pause.png</disabledon>:/Resources/play.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="volButton">
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>190</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-style:solid;
border-width:0px;
border-color:rgb(255,255,255);
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resources/volume.png</normaloff>:/Resources/volume.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mminButton">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>3</y>
     <width>29</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-style:solid;
border-width:0px;
border-color:rgb(255,255,255);
background:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resources/minin.png</normaloff>:/Resources/minin.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="minButton">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>3</y>
     <width>29</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255,255,255);
border-style:solid;
border-width:0px;
border-color:rgb(255,255,255);
background:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resources/min.png</normaloff>:/Resources/min.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>3</y>
     <width>29</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-style:solid;
border-width:0px;
border-color:rgb(255,255,255);
background:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resources/close.png</normaloff>:/Resources/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="logoButton">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>2</y>
     <width>120</width>
     <height>48</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);
border-style:solid;
border-width:0px;
border-color:rgb(255,255,255);
background:transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Resources/logo.png</normaloff>:/Resources/logo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>120</width>
     <height>48</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>170</y>
     <width>35</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QSlider::add-page:vertical
     {     
        background-color: rgb(37, 168, 198); 
        width:4px;
     }
     QSlider::sub-page:vertical 
    {
        background-color: rgb(87, 97, 106); 
        width:4px;
     }
    QSlider::groove:vertical 
    {
        background:transparent;
        width:6px;
    }
    QSlider::handle:vertical 
    {
         height:-4px;
         width:12px;
         margin: -0 -4px; 
     }</string>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>iceLabel</class>
   <extends>QLabel</extends>
   <header location="global">icelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
